server.js

```diff
@@ -88,6 +88,13 @@ io.on("connection", client => {
 
   // TODO Create a subscription for this new client to some occupancy changes
   // TODO subscribe to realOccupancyChanges AND simulatedOccupancyChanges
+  // TODO subscribe to realOccupancyChanges AND simulatedOccupancyChanges
+  const sub = simulatedOccupancyChanges.subscribe(notifyClient);
+
+  client.on("disconnect", () => {
+    sub.unsubscribe();
+    console.log("Client disconnected");
+  });
```

Comment:
An alternative to this style is discussed in the [WebSockets Case Study](https://github.com/deanius/rx-helper/wiki/Case-Study:-Web-Sockets